iaso/api/common.py

```diff
@@ -510,3 +510,13 @@ def list(self, request):
         status_choices = self.get_status_choices()
         serializer = self.serializer(status_choices, many=True)
         return Response(serializer.data)
+
+
+class RoundingDecimalField(serializers.DecimalField):
+    """
+    As per the overridden method's documentation, this can be overridden to disable the validation.
+    The quantize method is called afterward and does the rounding.
+    """
+
+    def validate_precision(self, value):
+        return value
```

Comment:
Just wondering if we're not losing some other useful validation by completely overriding this (see https://github.com/encode/django-rest-framework/blob/543996711d323722a1017e376619ae462a726ada/rest_framework/fields.py#L1049-L1082)?

Reply:
I suggest implementing `validate_precision` like this instead https://github.com/BLSQ/iaso/pull/2130

Let me know if that's OK for you?

Reply:
Sure, I'm fine with that :)